### PR TITLE
Add Platform Builders

### PR DESCRIPTION
**Pull request by bpedroso**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @bpedroso, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 1f20c8a).  See below for details:

```
README.md: Company is listed out of order: "Platform Builders" (should be before "Platform.sh")
platformbuilders.md: Company title "Platform Builders" doesn't match filename (expected ~ "platform-builders.md")

2 problems detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @bpedroso, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit a8bcaa8).  See below for details:

```
README.md: Company is listed out of order: "Platform Builders" (should be before "Platform.sh")
README.md: Missing company profile for "Platform Builders", or broken link: "/company-profiles/platformbuilders.md"
platform-builders.md: No link to company profile from readme

3 problems detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @bpedroso, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 2b64e59). :heavy_check_mark:

File: README.md [hunk 1]
```diff
@@ -337,6 +337,7 @@ Name | Website | Region
 [PeopleDoc](/company-profiles/peopledoc.md) ⚠️️ | http://www.people-doc.com |
 [Percona](/company-profiles/percona.md) | https://www.percona.com | Worldwide
 [PicksCo](/company-profiles/picksco.md) | http://interchangehq.com/ | UTC-7 to UTC+1
+[Platform Builders](/company-profiles/platform-builders.md) | https://platformbuilders.io/ | Brazil and Portugal
```
Comment (dougaitken) [inline README.md]: The "Region" in this section should match the region in your company profile. 

You mention Worldwide there so if you hire people worldwide, please update this to match. Thanks!

Comment (bpedroso) [reply README.md]: Ok. Change the region of Platform Builders to Worldwide. Thanks.

**Comment by dougaitken** [comment]
Thanks @bdeproso!

**Pull request merged**